bin/client_docker/alpine/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.9.2-alpine AS builder
+
+# Install packages
+RUN apk add --update --no-cache bash g++ make \
+    && rm -rf /var/cache/apk/*
+
+# Compile application
+WORKDIR /go/src/github.com/mysterium/node
+ADD . .
+RUN bin/client_build
+
+
+
+FROM alpine:3.6
+MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
+
+# Install packages
+RUN apk add --update --no-cache bash iptables ca-certificates openvpn \
+    && rm -rf /var/cache/apk/*
+
+# Install application
+COPY bin/client_docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
From docker perspective it's not very efficient to put executable AND it's args into entry point, as it will be very hard to override params later. It should be like this:
ENTRYPOINT ["your exec and args which must be always present and never change"]
CMD["default additional args which can be customized when using docker run <image> <args> style"]
those could be tequila.port , address , discovery address etc.
Do we really even need docker-entrypoint.sh ? it hides important stuff, also uses env vars internally and it's hard to track what is going on

Reply:
Good ideas, will depreciate Docker environments variables in upcoming PRs